content/pages/about/technology-overview.md

```diff
@@ -162,20 +162,20 @@ __External Components__ - the following standalone components can be used with a
 
 __Clients__
 
-- [Adium](http://adiumx.com/)
-- [Gajim](http://gajim.org/)
+- [Adium](https://adiumx.com/)
+- [Gajim](https://gajim.org/)
 - [JWChat](http://blog.jwchat.org/jwchat/)
-- [mcabber](http://mcabber.com/)
-- [Pidgin](http://pidgin.im/)
+- [mcabber](https://mcabber.com/)
+- [Pidgin](https://pidgin.im/)
 - [Psi](http://www.psi-im.org/)
 
 __Libraries__
 
 - [AnyEvent:XMPP](http://www.ta-sa.org/projects/anyevent_xmpp.html) (Perl)
-- [gloox](http://camaya.net/gloox) (C++)
-- [jabber-net](http://code.google.com/p/jabber-net/) (.Net)
-- [libpurple](http://developer.pidgin.im/wiki/WhatIsLibpurple) (C)
-- [Smack](http://www.igniterealtime.org/projects/smack/index.jsp) (Java)
+- [gloox](https://camaya.net/gloox) (C++)
+- [jabber-net](https://code.google.com/p/jabber-net/) (.Net)
```

Comment:
Google Code is now just an archive. Does this project still exist? If so, this link is wrong, if not, it should be removed. I know that's not really related to this PR, but I don't much see the point of updating everything to explicitly use HTTPs if it's also broken or not useful.